code/modules/mob/living/carbon/human/species_types/shadowpeople.dm

```diff
@@ -61,7 +61,7 @@
 	return ..()
 
 /datum/species/shadow/nightmare/check_roundstart_eligible()
-	return FALSE
+	return .....()//skips the parent proc so is just the normal roundstart eligible (don't pass POSSIBLE)

```

Comment:
If you're trying to call species/check_roundstart_eligible I believe this should just be ...()

As well as for all the others

Reply:
Do not `.....()`

Reply:
what in the everloving hell is this, if you ever write this line of code it is a sign you are doing something massively stupid.

Add an override var